src/Findologic/Request/Handler/NavigationRequestHandler.php

```diff
@@ -106,7 +106,7 @@ public function handleRequest(ShopwareEvent $event): void
      * @throws ServiceNotAliveException
      * @throws UnknownCategoryException
      */
-    public function doRequest(ShopwareEvent $event, ?int $limit = null): Response
+    public function doRequest(ShopwareEvent $event, ?int $limit = null, ?bool $forceFilter = null): Response
```

Comment:
Without deep knowledge of the implementation, you do not know what the parameter `$forceFilter` is supposed to do. Please add this in the method docblock. Same goes for `SearchNavigationRequestHandler::doRequest` and `SearchRequestHandler::doRequest`.